torchgeo/datasets/vaihingen.py

```diff
@@ -0,0 +1,396 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Vaihingen dataset."""
+
+import os
+from typing import Any, Callable, Dict, Optional
+
+import matplotlib.pyplot as plt
+import pytorch_lightning as pl
+import rasterio
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+from torch.utils.data import DataLoader
+from torchvision.transforms import Compose
+
+from ..datasets.utils import dataset_split, draw_semantic_segmentation_masks
+from .geo import VisionDataset
+from .utils import check_integrity, extract_archive, rgb_to_mask
+
+
+class Vaihingen(VisionDataset):
+    """Vaihingen dataset.
```

Comment:
Same question as your other dataset, I think we'll want to make this name more specific